pyEpiabm/pyEpiabm/utility/covidsim_kernel.py

```diff
@@ -0,0 +1,31 @@
+#
+# Class of Covidsim-style gravity kernels
+#
+
+
+class Kernel:
+    """Class to create the gravity kernel used throughout
+    CovidSim in the spatial weighting.
```

Comment:
Might be worth citing it here, and certainly adding to the wiki with reference!

Reply:
That now looks good - but maybe have a link to the page where the detail is (that seems to go to the homepage?)